app/models/passenger.rb

```diff
@@ -0,0 +1,19 @@
+class Passenger < ApplicationRecord
+  has_many :trips, dependent: :nullify
+
+  def total_charged
+    trips_with_passenger_id = self.trips.where(passenger_id: id)
+
+    sum = 0
+
+    trips_with_passenger_id.each do |trip|
+      sum += trip.cost
+    end
```

Comment:
You can clean this up a little using the `sum` method:

```suggestion
    sum = self.trips.sum { |trip| trip.cost }
```